SECURITY.md

```diff
@@ -27,9 +27,9 @@ The following table provides the status of the Otter versions under support and
 
 | Otter version | Angular version | Support ends |
 |---------------|-----------------|--------------|
+| ^9.2.0        | ^16.0.0         | 2024-11-08   |
 | ^8.2.0        | ^15.0.0         | 2024-05-18   |
```

Comment:
```suggestion
| ^8.3.0        | ^15.0.0         | 2024-05-18   |
```

Reply:
^8.x

Reply:
don't we only support the last minor of each major ?